src/libraries/Microsoft.Extensions.Primitives/src/Extensions.cs

```diff
@@ -5,6 +5,9 @@
 
 namespace Microsoft.Extensions.Primitives
 {
+    /// <summary>
+    /// Extenstion methods for the <see cref="Primitives"/> namespace
```

Comment:
Just checking - did the `cref` resolve correctly in VS?

```suggestion
    /// Provides extensions methods for the <see cref="Primitives"/> namespace.
```

Reply:
Yep, if you prefer I could make it the entire `cref="Microsoft.Extenstions.Primitives"` for readability in source but either implementation shows the same preview in VS

Reply:
No need, I just wanted to make sure it was allowed for namespaces. Once you build this project, the generated intellisense xml file will automatically use the full name of that DocId.